src/components/common/ProgressBar/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export type Props = {
+  readonly percentage?: string
+  readonly activeColor?: string
+  readonly showLabel?: boolean
+}
+
+const Wrapper = styled.div<Props>`
+  display: flex;
+  align-items: center;
+
+  > div {
+    width: 16rem;
+    height: 0.6rem;
+    position: relative;
+    border-radius: 16rem;
+    background: ${({ theme }): string => theme.borderPrimary};
+  }
+
+  > div > span {
+    width: ${({ percentage }): string => percentage || '0'}%;
```

Comment:
How to keep the fallback out of this here? Can only be done in the parent component were we get the props from?

Reply:
You can define the props of the styled compoenent different to the component itself. See my other comment